lib/common/logging.c

```diff
@@ -129,9 +133,20 @@ crm_trigger_blackbox(int nsig)
 void
 crm_log_deinit(void)
 {
-    if (glib_log_default != NULL) {
-        g_log_set_default_handler(glib_log_default, NULL);
+    if (pcmk__log_id == 0) {
+        return;
     }
+
+    g_log_remove_handler(G_LOG_DOMAIN, pcmk__log_id);
+    pcmk__log_id = 0;
+    g_log_remove_handler("GLib", pcmk__glib_log_id);
+    pcmk__glib_log_id = 0;
+    g_log_remove_handler("GLib-GIO", pcmk__gio_log_id);
+    pcmk__gio_log_id = 0;
+    g_log_remove_handler("GModule", pcmk__gmodule_log_id);
+    pcmk__gmodule_log_id = 0;
+    g_log_remove_handler("GThread", pcmk__gthread_log_id);
+    pcmk__gthread_log_id = 0;
```

Comment:
We most likely will never use GModule or GThread, so we can drop those

Reply:
They are both potentially used by glib somewhere - gthread looks like it could be used by mainloop stuff, and gmodule is potentially used by gio.